terraform/modules/apps/content-store/main.tf

```diff
@@ -7,23 +7,27 @@ terraform {
   }
 }
 
+locals {
+  service_name_snake_case = replace("${var.service_name}", "-", "_")
+}
+
```

Comment:
I think it's better to just use the name as-is. This kind of thing tends to make it hard to trace where values are coming from (for example by breaking searchability of the codebase).